### PR TITLE
Update Documentation.md

### PR DESCRIPTION
**Pull request by randalld-aws**
Adding error events for CCP connectivity issues

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Documentation.md [hunk 1]
```diff
@@ -835,3 +835,65 @@ call `.withObject(o)` to add an arbitrary object (`o`) to the logs.
...
+Internal connection error occurred. To resolve, try the action again, or wait a minute and then retry. If the issue persists, contact support.
+
+### `ice_collection_timeout`
+Error connecting to the media service. To resolve, try the action again, or wait a minute and then retry. If the issue persists, contact support.
+
+### `user_busy_error`
+Agent has the CCP running in 2 distinct browsers at the same time, such as Chrome and Firefox. Use only one browser at a time to log in to the CCP.
+
+### `webrtc_error`
+An issue occurred due to either using an unsupported browser, or a required port/protocol is now open, such as not allowing UDP on port 443. To resolve, confirm that the agent is using a supported browser,a nd that all traffic is allowed for all required ports and protocols. See [CCP Networking](https://docs.aws.amazon.com/connect/latest/adminguide/troubleshooting.html#ccp-networking) and [Phone Settings](https://docs.aws.amazon.com/connect/latest/userguide/agentconsole-guide.html#phone-settings) for more information.
```
Comment (mschersten) [inline Documentation.md]: This should be "is not open" instead of "is now open"?

Comment (mschersten) [reply Documentation.md]: Also a typo at "browser,a nd"

Comment (randalld-aws) [reply Documentation.md]: thank you!

File: Documentation.md [hunk 2]
```diff
@@ -835,3 +835,68 @@ call `.withObject(o)` to add an arbitrary object (`o`) to the logs.
 
 Finally, you can trigger the logs to be downloaded to the agent's machine in JSON form by calling `connect.getLog().download()`.
 
+## CCP Error Logging
+The following errors are related to connectivity in the CCP. These errors are logged in the CCP logs when they occur.
+
+### `unsupported_browser` 
+Agent is using an unsupported browser. Only the latest 3 versions of Chrome or Firefox is supported. Upgarde the agent's browser to resolve this error. See [Supported browsers](https://docs.aws.amazon.com/connect/latest/adminguide/what-is-amazon-connect.html#browsers)  for more information.
```
Comment (jagadeeshaby) [inline Documentation.md]: upgrade - misspelled

**Pull request merged**